sdk/storage/Azure.Storage.Common/src/Shared/Constants.cs

```diff
@@ -205,10 +205,21 @@ internal static class File
             public const int MaxFileUpdateRange = 4 * MB;
             public const string FileTimeFormat = "yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffffff'Z'";
 
+            internal static class Lease
+            {
+                /// <summary>
+                /// Lease Duration is set as infinite when passed -1.
+                /// </summary>
+                public const int InfiniteLeaseDuration = -1;
+            }
+
             internal static class Errors
             {
                 public const string ShareUsageBytesOverflow =
                     "ShareUsageBytes exceeds int.MaxValue. Use ShareUsageInBytes instead.";
+
+                public const string LeaseNotPresentWithFileOperation =
+                    "LeaseNotPresentWithFileOperation";
```

Comment:
Is there a constant in `FileErrorCode` for this?  If not, we should update the swagger.

Reply:
(Can defer)

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/9856